dotnet/src/SemanticKernel.UnitTests/Planning/PlanTests.cs

```diff
@@ -92,7 +92,7 @@ public async Task CanExecutePlanWithPlanStepAsync()
 
         var mockFunction = new Mock<ISKFunction>();
         mockFunction.Setup(x => x.InvokeAsync(It.IsAny<SKContext>(), null, It.IsAny<CancellationToken>()))
-            .Callback<SKContext, CompleteRequestSettings, CancellationToken>((c, s, ct) =>
+            .Callback<SKContext, dynamic, CancellationToken>((c, s, ct) =>
```

Comment:
In these similar mocks I see the usage of `dynamic`, `object` and `AIRequestSettings`. I'm wondering what the reason is, and would it be possible to use just `AIRequestSettings` everywhere?